.github/workflows/benchmarks.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           else
               echo "HAS_SECRET=0" >> $GITHUB_ENV
           fi
-      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@main
```

Comment:
FYI: Fix https://github.com/CyberAgentAILab/cmaes/runs/5883904567?check_suite_focus=true

Reply:
Hmm... The usage of this action is changed recently.
https://github.com/google-github-actions/setup-gcloud